app/models/suggestion.rb

```diff
@@ -0,0 +1,5 @@
+class Suggestion < ApplicationRecord
+  validates :author, :link, :title, :publisher, :year, :editorial, presence: true
+
+  belongs_to :user
```

Comment:
Rails 5 makes all `belongs_to` relations mandatory, lets keep this as optional `belongs_to :user, optional: true`.
This way an unlogged user can create BookSuggestions